setup.py

```diff
@@ -148,6 +130,7 @@ def __init__(self, name, extra_cmake_args=[]):
 # They're built along with the extension, in the order listed.
 AWS_LIBS = []
 if sys.platform != 'darwin' and sys.platform != 'win32':
+    AWS_LIBS.append(AwsLib('aws-lc'))
```

Comment:
It's not merged yet, but there's an outstanding PR for aws-lc that will let us turn off building the parts of aws-lc that we don't need: https://github.com/awslabs/aws-lc/pull/105

Anyway, add this extra cmake arg to aws-lc. When that PR gets merged it will eventually speed us up

`AWS_LIBS.append(AwsLib('aws-lc', ['BUILD_LIBSSL=OFF']))`

Reply:
+1